FeelYourSelfLead/Task_45/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Показать числа Фибоначчи
+
+int a = 0;
+int b = 1;
+int c = 1;
+Console.Write($"0,1,1");
+
+for (int i = 0; i < 10; i++)
```

Comment:
why do you need **for loop** here if you decided to use this simple approach for this task?